src/services/expenses.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+let expenses = [];
+
+class ExpensesService {
+  setInitialExpenses() {
+    expenses = [];
+  }
```

Comment:
If you create your service as separate class, then you can store expenses inside it. Initialise em inside constructor.